packages/terra-action-footer/src/ActionFooter.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import 'terra-base/lib/baseStyles';
+import ActionFooterContainer from './_ActionFooterContainer';
+import styles from './ActionFooter.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+   * Actions to be displayed in the start socket
+   */
+  start: PropTypes.node,
+  /**
+   * Actions to be displayed in the end socket
+   */
+  end: PropTypes.node,
+};
+
+const defaultProps = {
+  start: undefined,
+  end: undefined,
+};
+
+const ActionFooter = ({
+  start,
+  end,
+  ...customProps
+}) => {
+  const isEmpty = !start && !end;
+
+  return (isEmpty ?
+    <ActionFooterContainer {...customProps} className={cx(customProps.className)} />
+    :
+    <ActionFooterContainer {...customProps} className={cx(customProps.className)}>
+      { start && <div className={cx('socket')}>{start}</div> }
+      <div className={cx('empty-center')} />
+      { end && <div className={cx('socket')}>{end}</div> }
+    </ActionFooterContainer>
+  );
+};
```

Comment:
The formatting here is up to your preference. Possible other ways to format these could be:

```
const ActionFooter = ({
  start,
  end,
  ...customProps
}) => {
  if (!start && !end) {
    return <ActionFooterContainer {...customProps} className={cx(customProps.className)} />;
  }

  return (
    <ActionFooterContainer {...customProps} className={cx(customProps.className)}>
      { start && <div className={cx('socket')}>{start}</div> }
      <div className={cx('empty-center')} />
      { end && <div className={cx('socket')}>{end}</div> }
    </ActionFooterContainer>
  );
};
```
or
```
const ActionFooter = ({
  start,
  end,
  ...customProps
}) => (
  <ActionFooterContainer {...customProps} className={cx(customProps.className)}>
    { start && <div className={cx('socket')}>{start}</div> }
    { (start || end) && <div className={cx('empty-center')} />}
    { end && <div className={cx('socket')}>{end}</div> }
  </ActionFooterContainer>
);
```

Reply:
Single, implicit return, I like the second one 🙌

Reply:
I _do_ like the second option. I feel like this could get _real opinionated real quick_ though. Then again, I'm already using a ternary.. 🤔